contents/blog/posthog-marketing.md

```diff
@@ -0,0 +1,219 @@
+
+### Core KPIs
+
+This dashboard is setup in a grid with two insights per KPI – that's key performance indicator if don't speak "marketing". One is the total number with 'compare to previous period' enabled; the second is the same metric as a weekly trend. This gives us an at-a-glance view of the top-level number, and context around peaks and troughs.
+
+![marketing-dashboard](../images/blog/posthog-marketing/marketing-dashboard.png)
+
+We look at the last 90 days by default, but you can change the date range of the whole dashboard whenever you like. We add annotations to denote important events – e.g. changes to landing page designs, or a blog article going viral.
+
+![annotation](../images/blog/posthog-marketing/annotation.png)
+
+(Seriously, people, **use annotations!** It's way easier than trying to remember the date you redesigned your pricing page six months ago.)
+
+### Circuit breakers
+
+We like to iterate our website quickly, which often means pushing new designs live without testing. As a fallback, we track various "circuit breaker" metrics on our marketing dashboard, such as the conversion rate on key landing pages.
+
+In July, for example, we discovered a tracking bug between 'posthog.com' and 'app.posthog.com' which caused an obvious drop in conversions. Then, in August, we introduced a new pricing page experience. 
+
+![pricing trend](../images/blog/posthog-marketing/pricing-trend.png)
+
+As you can see (above), our first iteration was a pretty dramatic failure. Users visiting the pricing page and "showing intent" (we'll get into what this means later) declined from ~25% down to ~10%. It was also obvious from watching session recordings that our new pricing wizard was simply confusing people.
+
+This lead our Website & Docs team to iterate several new versions, which they then tested using feature flags and experiments. Conversions have since returned to normal and we've simplified our pricing as a result.
+
+> 💡 **PostHog Tip:** Try using filters on dashboards! You can apply global filters to any dashboard based on event properties, person properties, feature flags, and cohorts. This is a great way to drill down into specific audiences (e.g. users by country, city, continent).
+
+## Website dashboard
+
+One level below we have our main website dashboard. This is less focused on conversion metrics and more on general traffic trends across posthog.com. Here, we're looking at typical analytics things like:
+
+- Total website users
+- Organic SEO users
+- Unique users reading the blog
+- The most read pages in each website section
+- A map view showing website users by country
+
+This is all pretty straightforward stuff, but it's still very useful to us as a marketing team. Recently, for example, we identified a specific tutorial that was getting loads of SEO traffic but was in serious need of updating.
```

Comment:
```suggestion
This is all straightforward stuff, but it's still useful to us as a marketing team. Recently, for example, we identified a tutorial that was getting loads of SEO traffic but was in serious need of updating.
```

Reply:
> > Love it! Agree on title - what about 'if you're only using PostHog for your product, you're missing out' or 'the ultimate guide to using PostHog for marketing analytics' or something in that ballpark?
> 
> Think it would be best to keep marketing up front in the title. 'The ultimate guide to marketing analytics in PostHog' might work if you aren't happy with current title.

I think we're good with the title now (at least I am). Context: the title was just 'How we use PostHog for marketing' or something like that.